packages/opentelemetry-tracing/src/export/ConsoleSpanExporter.ts

```diff
@@ -17,28 +17,65 @@
 import { SpanExporter } from './SpanExporter';
 import { ReadableSpan } from './ReadableSpan';
 import { ExportResult } from '@opentelemetry/base';
-import { hrTimeToMilliseconds } from '@opentelemetry/core';
+import { hrTimeToMicroseconds } from '@opentelemetry/core';
 
 /**
  * This is implementation of {@link SpanExporter} that prints spans to the
  * console. This class can be used for diagnostic purposes.
  */
 export class ConsoleSpanExporter implements SpanExporter {
+  /**
```

Comment:
Nit/optional: what would you think about making this comment a simple single line comment, so `/** Writes spans to the console */`?

I personally find the param comments don't add much since they are already in the type signature, and the "Export spans" doesn't add much beyond what the function name and params give.